### ISSUE TITLE
How much potential revenue can you mk running the bot 24/7?

### ISSUE THREAD

**Issue by DartMouthWrld** [BhaskarPanja93/View-Bot-Client#41]

**Comment by KevinGeLe** [comment]
Each view takes 2 minutes and 40 seconds on average for me. Every view pays between 2-3 cent. So to calculate the amount of views in a single day we divide the amount of minutes in a single day, with how long a view takes so (1440/2,66=541) per VM. That we can time with how much each view is worth (541*2,5=1353) cents per day or 13,53$. Then ofcourse its easily scaleable. The last time I tried to use my own Linkvertise bot (not this one,) I got instant banned when I tried to payout. Now I havent gotten banned on Linkvertise using this bot, but they have said in their support that they cancelled my payout and they will re-evaluate my payout. So havent gotten my money yet and is still waiting till weekend is over for a reply. 

Maybe we have to add so some of the views doesnt go through, because under normal circumstances, you wont get a 1:1 ratio between impressions and clicks. So having it something like 3:1 will almost certainly remove that and also having more ip's, so that it looks likes its coming from a lot of different places.

TL:DR
I'm getting 13$ a day per VM

@qwakup2314 
@BhaskarPanja93

**Comment by BhaskarPanja93** [comment]
okay, the last time i used the bot, each view was giving me around 0.11 dollars. So i am not sure of the calculations. But pretty sure its generating a lot, enough to get the account banned forever :) So dont take the risk, and go slow :)

**Comment by DartMouthWrld** [comment]
How can I scale up? My computer can't take more VMs...
